vhdbuilder/packer/install-dependencies.sh

```diff
@@ -394,6 +394,22 @@ while IFS= read -r p; do
         echo "  - kubernetes-binaries version ${version}" >> ${VHD_LOGS_FILEPATH}
       done
       ;;
+    "kubelet")
+      for version in ${PACKAGE_VERSIONS[@]}; do
+        if [ "${OS}" = "${UBUNTU_OS_NAME}" ] || isMarinerOrAzureLinux "$OS"; then
```

Comment:
humm I'm realizing now, I'm not sure what we are doing for Windows...  We should add a US under the Epic to keep track of this work